"kardianos/osext" is obsolete on go-1.8+

The following patch allows to remove redundant `github.com/kardianos/osext` dependency:

~~~~
--- a/panicwrap.go
+++ b/panicwrap.go
@@ -20,9 +20,8 @@
        "sync/atomic"
        "syscall"
        "time"

-       "github.com/kardianos/osext"
 )

 const (
        DEFAULT_COOKIE_KEY = "cccf35992f8f3cd8d1d28f0109dd953e26664531"
@@ -117,9 +116,9 @@
                return -1, nil
        }

        // Get the path to our current executable
-       exePath, err := osext.Executable()
+       exePath, err := os.Executable()
        if err != nil {
                return -1, err
        }

~~~~

I would implore the package owners here to please remove this dependency. I get this error:

```
vendor/github.com/bugsnag/panicwrap/monitor.go:6:2: cannot find package "github.com/kardianos/osext" in any of:
	/Users/max/go/src/echo_influx_interface/vendor/github.com/kardianos/osext (vendor tree)
	/usr/local/Cellar/go/1.11/libexec/src/github.com/kardianos/osext (from $GOROOT)
	/Users/max/go/src/github.com/kardianos/osext (from $GOPATH)
2018/11/21 15:15:38 exit status 1
```

--- a/go.mod
+++ b/go.mod
@@ -1,3 +1,1 @@
 module github.com/mitchellh/panicwrap
-
-require github.com/kardianos/osext v0.0.0-20170510131534-ae77be60afb1

--- a/panicwrap.go
+++ b/panicwrap.go
@@ -20,8 +20,6 @@ import (
 	"sync/atomic"
 	"syscall"
 	"time"
-
-	"github.com/kardianos/osext"
 )
 
 const (
@@ -118,7 +116,7 @@ func Wrap(c *WrapConfig) (int, error) {
 	}
 
 	// Get the path to our current executable
-	exePath, err := osext.Executable()
+	exePath, err := os.Executable()
 	if err != nil {
 		return -1, err
 	}

